external/singleapplication/singleapplication.cpp

```diff
@@ -434,6 +436,25 @@ SingleApplication::SingleApplication(int& argc,
 
     d->connectToPrimary(timeout, SingleApplicationPrivate::NewInstance);
     delete d;
+
+    // show message box with inforation that Flameshot is already launched
```

Comment:
I see that the external singleapplication implementation is modified to pop up some flameshot-specific notification. This should either be unacceptable or filed as a bug (issue) and be tracked via github issue page.

This is due to we are still targeting on splitting the embedded singleapplication out and such modification will be lost if we switch to external singleapplication libs.

The ideal solution is to put such function in the main flameshot codebase. I am not sure if it is possible though.

Reply:
In general I agree with you, but I hadn't enough time to handle it in "True" way. It was done before the `Flameshot` reincarnation :)
I'm ok if we'll remove it.